test/unit/axe-injector.js

```diff
@@ -143,7 +143,7 @@ describe('AxeInjector', () => {
       let didReturnIframes = false;
       const driver = new MockWebDriver({
         findElements(selector) {
-          assert.equal(selector.tagName, 'iframe');
+          assert.equal(selector.value, 'iframe');
```

Comment:
You should also do an `assert.equal(selector.using, 'tagName');` for this to be equivalent.